dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -255,7 +340,31 @@ def _parse_sheet(self, sheet):
                     )
                 continue
 
+
+
+            # Booster data not available before November 2021.
+            if  self.publish_date < datetime.date(2021, 11, 1) \
+                and (sig in ["booster dose since", "booster doses administered"]) :
+                self.dfs[(sheet.level, sig)] = pd.DataFrame(
+                        columns = ["geo_id", "timestamp", "val", \
+                            "se", "sample_size", "publish_date"]
+                    )
+                continue
+
+            # Booster and weekly doses administered not available below the state level.
+            if ((sheet.level != "hhs" and sheet.level != "state") \
+                and (sig in ["doses administered", \
+                 "booster doses administered", "booster dose since"])):
+                self.dfs[(sheet.level, sig)] = pd.DataFrame(
+                        columns = ["geo_id", "timestamp", "val", \
+                            "se", "sample_size", "publish_date"]
+                    )
+                continue
+
             sig_select = [s for s in select if s[-1].find(sig) >= 0]
+
+            if sig == "doses administered":
+                sig_select = [s for s in select if s[-1].startswith(sig)]
```

Comment:
Here, `startswith` returns a boolean (True/False), which when converted to int gives 0/1 (which are >=0, so not as informative).

Reply:
```suggestion
            sig_select = [s for s in select if s[-1].find(sig) >= 0]

            # Since "doses administered" is a substring of another desired header,
            # "booster doses administered", we need to more strictly check if "doses administered"
            # occurs at the beginning of a header to find the correct match.
            if sig == "doses administered":
                sig_select = [s for s in select if s[-1].startswith(sig)]
```